lib/ExpensiMark.js

```diff
@@ -61,8 +61,8 @@ export default class ExpensiMark {
              */
             {
                 name: 'email',
-                regex: /\[([\w\\s\\d!?&#;]+)\]\(([a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z]+?(\.[a-zA-Z]+)+)\)/gim,
-                replacement: '<a href="mailto:$2">$1</a>',
+                regex: /\[([^[\]]*)\]\(([a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z]+?(\.[a-zA-Z]+)+)\)/gim,
+                replacement: (match, g1, g2) => `<a href="mailto:${g2}">${g1.trim()}</a>`,
```

Comment:
Can we update this replacement function to something like - 
`g1.trim() ? '<a href="mailto:${g2}">${g1.trim()}</a>' : ''`
Thoughts? cc @mananjadhav

Reply:
Agreed with @aimane-chnaif